### PR TITLE
google maps update

### PR DESCRIPTION
**Pull request by jastring1**
Added labels to the markers in google maps and created a column in the dynamic table to show the labels.

### CODE REVIEW

File: assets/javascript/logic.js [hunk 1]
```diff
@@ -64,18 +64,21 @@ function zomatoGetRestaurants(a) {
         success: function (data) {
             console.log(data);
             var z = data.restaurants;
+            var labels = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+            var labelIndex = 0;
             $("#table-head").empty();
```
Comment (tcoleman1) [inline assets/javascript/logic.js:69]: I've already started on the js file. Did you do a pull .. ?

Comment (jastring1) [reply assets/javascript/logic.js:69]: I submitted a PR about 2 hours ago.

Comment (jastring1) [reply assets/javascript/logic.js:69]: Once it is approved and merged you can pull it to your branch and it should not affect anything that you have been working on.

Comment (jastring1) [reply assets/javascript/logic.js:69]: I also do not see a branch for you on the repository so make sure you are working on your own branch

**Review by toneyhiggins2 — APPROVED** [review]

**Pull request merged**